content/modals/GB/short_term_xo.json

```
@@ -0,0 +1,51 @@
+{
+    "meta": {
+        "product": "PAY_LATER_SHORT_TERM",
+        "periodicPayment": "{formattedPeriodicPayment}",
+        "minAmount": "{minAmount}",
+        "maxAmount": "{maxAmount}",
+        "qualifying": "{qualifying_offer}",
+        "amount": "{transaction_amount}",
+        "apr": "{apr}",
+        "preapproved": true,
```

```suggestion
        "preapproved": false,
```
For the sake of local testing and just having the regular non-preapproval checkout version show up first, let's just have this default to `false`.

---

i agree